app/javascript/packs/admin.js

```diff
@@ -12,3 +12,5 @@ require('admin/magnific-popup')
 require('admin/nanoscroller')
 require('admin/theme')
 require('admin/sidebar')
+require('admin/bootstrap-tagsinput')

```

Comment:
move this line higher, theme and sidebar are not libraries but the others are

Reply:
ok